mkdocs.yml

```diff
@@ -45,6 +45,7 @@ nav:
       - Knowledge Base repo file: sc-tools/kb_repo_file.md
       - Websocket Server: sc-tools/sc_server.md
       - Sc-machine Runner: sc-tools/sc_machine.md
+      - Codegen removal: sc-tools/codegen_removal.md
```

Comment:
I think we should emphasise that this is a script to migrate from the old API to the new one. I propose to name this script `migrate_to_new_agent_api` and note that it replaces the old API using codegeneration with a new API